.buildkite/build-static-release.sh

```diff
@@ -28,10 +32,25 @@ esac
 
 echo will run with $CONFIG_OPTS
 
-./bazel build //main:sorbet --strip=always $CONFIG_OPTS
+case "$platform" in
+  darwin-x86_64|darwin-arm64)
+    ./bazel build //main:sorbet --strip=always $CONFIG_OPTS
+    cp bazel-bin/main/sorbet sorbet_x86_64
+
+    ./bazel build //main:sorbet --strip=always $CONFIG_OPTS --config=release-mac-arm64-cross
+    cp bazel-bin/main/sorbet sorbet_arm64
+
+    lipo -create -output sorbet_bin sorbet_x86_64 sorbet_arm64
```

Comment:
The bazel gymnastics for compiling for multiple architectures simultaneously can be done, but I think that would a) (definitely) be followup work and b) it's unlikely that it would be run anywhere outside of master builds.